README.md

```diff
@@ -77,11 +75,70 @@ app := goshopify.App{
 // Create a new API client
 client := goshopify.NewClient(app, "shopname", "token")
 
-// Fetch some data.
-products := client.Product.List()
+// Fetch the number of products.
+numProducts, err := client.Product.Count(nil)
+```
+
+#### Query options
+
+Most API functions take an options `interface{}` as parameter. You can use one
+from the library or create your own. For example, to fetch the number of
+products created after January 1, 2016, you can do:
+
+```go
+// Create standard CountOptions
+date := time.Date(2016, time.January, 1, 0, 0, 0, 0, time.UTC)
+options := goshopify.CountOptions{createdAtMin: date}
+
+// Use the options when calling the API.
+numProducts, err := client.Product.Count(options)
+```
+
+The options are parsed with Google's
+[go-querystring](https://github.com/google/go-querystring) library so you can
+use custom options like this:
+
+```go
+// Create custom options for the orders.
+// Notice the `url:"status"` tag
+options := struct {
+    Status string `url:"status"`
+}{"any"}
+
+// Fetch the order count for orders with status="any"
+orderCount, err := client.Order.Count(options)
 ```
 
+#### Using your own models
+
 Not all endpoints are implemented right now. In those case, feel free to
 implement them and make a PR, or you can create your own struct for the data
 and use `NewRequest` with the API client. This is how the existing endpoints
 are implemented.
+
+For example, let's say you want to fetch webhooks. There's a helper function
+`Get` specifically for fetching stuff so this will work:
+
+```go
+// Declare a model for the webhook
+type Webhook struct {
+    ID int         `json:"id"`
+    Address string `json:"address"`
+}
+
+// Declare a model for the resource root.
+type WebhooksResource struct {
+    Webhooks []Webhook `json:"webhooks"`
+}
+
+func FetchWebhooks() ([]Webhook, error) {
+	path := "admin/webhooks.json"
+	resource := new(WebhooksResoure)
+    client := goshopify.NewClient(app, "shopname", "token")
+
+    // resource gets modified when calling Get
```

Comment:
Some of it is copy-pasted from the actual code which explains the tabs. I'll convert it to spaces for the README :)